tests/adcc/cn_adc2/input.dat

```diff
@@ -0,0 +1,31 @@
+#! UADC(2)/cc-pvdz calculation of 5 states for the cyanide radical
```

Comment:
exceedingly trivial point, but `_` have been banned from test names in favor of `-`. I can always fix it up later.

Reply:
Sure, no problem. I fixed that.